tutorials/Tutorial11_Pipelines.py

```diff
@@ -201,6 +201,27 @@ def run(self, query):
     print_answers(res_2, details="minimum")
 
 
+    print("#######################")
+    print("# Debugging Pipelines #")
+    print("#######################")
+    # You can print out debug information from nodes in your pipelines in two different ways.
+
+    # 1) You can set the `debug` attribute of a given node.
+    es_retriever.debug = True
+
+    # 2) You can provide `debug` as a parameter when running your pipeline
+    result = p_classifier.run(
+        query="Who is the father of Arya Stark?",
+        params={
+            "ESRetriever": {
+                "debug": True
+            }
+        }
+    )
+
```

Comment:
We can also add a method 3: using `debug` as a global parameter

```python
    # 3) You can provide `debug` as a global parameter when running your pipeline
    result = p_classifier.run(
        query="Who is the father of Arya Stark?",
        params={
            "debug": True
        }
    )
```

Reply:
Great thanks for the tip!